core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -153,6 +154,18 @@ public WalWriter(
         }
     }
 
+
+    public long renameTable(CharSequence oldName, String newTableName) {
+        if (!Chars.equals(oldName, tableToken.getTableName())) {
```

Comment:
we need a test to assert case-insensitivity

Reply:
done, converted to case insensitive check, added tests